Makefile

```diff
@@ -39,7 +39,8 @@ answers:
 
 # -q flag is a workaround for jcapi-python lib not being in pypi (issue #9)
 package: clean docs
-	python setup.py -q sdist bdist_wheel
+        # pypi doesn't support bdist_wheel due to direct dependency on github
+	python setup.py -q sdist
```

Comment:
We can't because pypi doesn't support it and there's no indication that it's gonna change.  In any case it doesn't matter.  Even if we are able to publish the wheels package the installation from pypi will still be broken without a fix to issue #9.  Remember that we are only publishing to pypi to promote this package not for the purpose of installation.  

here's where i got the idea for removing wheels.. https://github.com/BaderLab/saber/issues/35#issuecomment-456120089